### PR TITLE
Feature/multiple exposure 

### PR DESCRIPTION
**Pull request by timarney**
https://trello.com/c/HqyhF3aK/806-people-can-receive-notifications-when-in-the-exposed-state

https://www.figma.com/file/0pLawavG1fnBTXSYn4pdKZ/Exposure-notification?node-id=13538%3A14

### CODE REVIEW

**Comment by timarney** [comment]
Testing on iOS 12.5 2 phones that had existing exposures (showing Jan 7th) but have been turned off for likely 5 days - when I updated the app the Notification date is empty.

File: src/services/ExposureNotificationService/ExposureNotificationService.ts [hunk 1]
```diff
@@ -809,7 +809,7 @@ export class ExposureNotificationService {
       const exposureHistory = _exposureHistory.split(',').map(x => Number(x));
       this.exposureHistory.set(exposureHistory);
     } catch (error) {
-      captureMessage('Unable to load EXPOSURE_HISTORY from secure storage', error);
+      captureMessage('No EXPOSURE_HISTORY found');
```
Comment (timarney) [inline src/services/ExposureNotificationService/ExposureNotificationService.ts:812]: Should change this to log.debug

File: src/services/ExposureNotificationService/ExposureNotificationService.ts [hunk 2]
```diff
@@ -748,11 +800,79 @@ export class ExposureNotificationService {
...
   }
 
+  private async loadExposureHistory() {
+    try {
+      const _exposureHistory = await this.secureStorage.get(EXPOSURE_HISTORY);
+      if (!_exposureHistory) {
+        log.debug({message: "'Unable to retrieve EXPOSURE_HISTORY"});
+        return;
+      }
+      const exposureHistory = _exposureHistory.split(',').map(x => Number(x));
```
Comment (smcmurtry) [inline src/services/ExposureNotificationService/ExposureNotificationService.ts:868]: make this a utility so it is more testable

**Comment by timarney** [comment]
Reviewed on a call with @smcmurtry - looking good.

File: src/screens/datasharing/components/BaseTekUploadView.tsx [hunk 3]
```diff
@@ -83,7 +83,7 @@ export const BaseTekUploadView = ({
       setIsUploading(false);
       onError(error);
     }
-  }, [contagiousDateInfo, fetchAndSubmitKeys, onError, onSuccess]);
+  }, [contagiousDateInfo, fetchAndSubmitKeys, onError, onSuccess, setIsUploading]);
```
Comment (smcmurtry) [inline src/screens/datasharing/components/BaseTekUploadView.tsx:86]: `yarn lint --fix` caught this.

**Review by dsamojlenko — APPROVED** [review]

**Pull request merged**